openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/common/ApiTests.java

```diff
@@ -0,0 +1,63 @@
+package org.openmetadata.catalog.selenium.pages.common;
+
+import static io.restassured.RestAssured.*;
+import static io.restassured.RestAssured.given;
+
+import com.google.gson.JsonParseException;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Test;
+
+public class ApiTests {
+
+  @Test
+  public void changeServiceDescription() throws JsonParseException {
+    try {
+      RestAssured.baseURI = "http://localhost:8585";
+      String response =
+          given()
+              .log()
+              .all()
+              .contentType(ContentType.JSON)
+              .body(PayLoad.changeDescriptionServices())
+              .when()
+              .put("api/v1/services/databaseServices")
+              .then()
+              .assertThat()
+              .extract()
+              .response()
+              .getBody()
+              .asString();
+      System.out.println("Updated Response=" + response);
```

Comment:
Remove println and use LOG?

Comment applies to other instances of this in this PR